corehq/apps/saved_reports/models.py

```diff
@@ -66,6 +66,7 @@
 from smtplib import SMTPSenderRefused
 
 from .logging import ScheduledReportLogger
+from ...util.quickcache import quickcache
```

Comment:
Could you make this an absolute import? more than one level of dots gets too confusing.